backend/ChessClubRegistration/src/main/java/com/marius/lt/ChessClubRegistration/controllers/ChessPlayerController.java

```diff
@@ -0,0 +1,73 @@
+package com.marius.lt.ChessClubRegistration.controllers;
+
+import com.marius.lt.ChessClubRegistration.converters.ChessPlayerConverter;
+import com.marius.lt.ChessClubRegistration.dtos.ChessPlayerDTO;
+import com.marius.lt.ChessClubRegistration.dtos.ChessPlayerEditDTO;
+import com.marius.lt.ChessClubRegistration.dtos.ChessPlayerPayloadDTO;
+import com.marius.lt.ChessClubRegistration.entities.ChessPlayer;
+import com.marius.lt.ChessClubRegistration.exceptions.ErrorDetails;
+import com.marius.lt.ChessClubRegistration.services.ChessPlayerService;
+import jakarta.persistence.NoResultException;
+import jakarta.validation.Valid;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Objects;
+
+@AllArgsConstructor
+@RestController
+@CrossOrigin
+@RequestMapping("/chess_player")
```

Comment:
Rekomenduociau sita resursa vadinti tiesiog zaideju `Player`, kadangi esame sachmatu klubo kontekste, naturalu, kad `Player` esantys jame - bus sachmatu klubo zaidejai.  Supaprastetu ir endpointu adresai ir lentu pavadinimai. O jei vis del to norima islaikyti chess player pavadinima, tada URI adresa reiketu vadint ne snake_case formatu, o kebab-case. Norint sudarineti adresus is daugiau nei vieno zodzio, snake_case yra nerekomenduotinas, nes snake case yra laikomas tarsi vienas zodis (pvz 1_000_000 -> yra milijonas, o ne 1 trys nuliai trys nuliai), o kebab-case formatu parasyti zodziai -> traktuojami kaip atskiri.

Pabandyk double clickint situos zodzius:
`chess_player` -> double clickinus parenka visa zodi
`chess-player` -> double clickinus parenka kiekviena zodi atskirai

Tai yra aktualu web crawleriams kurie skaitys tavo puslapi ir pagal rasta informacija ji indeskuos google ir panasiai :)